app/src/main/java/org/oppia/app/player/state/StateFragment.kt

```diff
@@ -9,18 +9,41 @@ import org.oppia.app.fragment.InjectableFragment
 import org.oppia.app.player.audio.CellularDataInterface
 import javax.inject.Inject
 
+private const val KEY_DIGIT_ID = "DIGIT_ID"
+private const val KEY_FETCHED_ID = "FETCHED_ID"
+
 /** Fragment that represents the current state of an exploration. */
 class StateFragment : InjectableFragment(), CellularDataInterface {
 
-  @Inject lateinit var stateFragmentPresenter: StateFragmentPresenter
+  @Inject
+  lateinit var stateFragmentPresenter: StateFragmentPresenter
 
   override fun onAttach(context: Context?) {
     super.onAttach(context)
     fragmentComponent.inject(this)
   }
 
-  override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? = stateFragmentPresenter.handleCreateView(inflater, container)
-  override fun enableAudioWhileOnCellular(saveUserChoice: Boolean) = stateFragmentPresenter.handleEnableAudio(saveUserChoice)
-  override fun disableAudioWhileOnCellular(saveUserChoice: Boolean) = stateFragmentPresenter.handleDisableAudio(saveUserChoice)
+  override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
+    var digit: String = ""
```

Comment:
Tyring making this lateinit.
Suggestion:
`private lateinit val digit: String`

Reply:
Resolved